cpp/src/Ice/OutgoingResponse.cpp

```diff
@@ -77,6 +76,8 @@ namespace
                 rfe.operation = current.operation;
             }
 
+            exceptionMessage = rfe.what();
```

Comment:
Moved here for consistency with the other language mappings.

We want to compute the message (what() in C++) after fixing up the identity/facet/operation.